yt/frontends/gadget_fof/data_structures.py

```diff
@@ -458,6 +458,11 @@ def __init__(self, ds, dataset_type="gadget_fof_halo_hdf5"):
             self.real_ds.parameter_filename, dataset_type
         )
 
+    @classmethod
```

Comment:
Is it still true that this can't be auto-loaded?  @brittonsmith I thought you had implemented an auto-detection for this?